src/Python/qsharp/azure.py

```diff
@@ -0,0 +1,47 @@
+#!/bin/env python
+# -*- coding: utf-8 -*-
+##
+# azure.py: enables using Q# quantum execution on Azure Quantum from Python.
+##
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+##
+
+## IMPORTS ##
+
+import qsharp
+import json
+import typing
+from typing import List, Dict, Callable, Any
+
+from qsharp.serialization import map_tuples
+from typing import List, Tuple, Dict, Iterable
+from enum import Enum
+
+## LOGGING ##
+
+import logging
+logger = logging.getLogger(__name__)
+
+## EXPORTS ##
+
+__all__ = [
+    'connect',
+    'target',
+    'submit',
+    'status'
+]
+
+## FUNCTIONS ##
+
+def connect(**params) -> Any:
```

Comment:
I think this makes sense for now, but we will likely want to narrow this from `Any` before release.

Reply:
Yes, as Efrat also pointed out, we need to figure out what information these magic commands should return as part of the `ExecutionResult`.

---
In reply to: [427478579](https://github.com/microsoft/iqsharp/pull/144#discussion_r427478579) [](ancestors = 427478579)